protected/humhub/libs/DynamicConfig.php

```diff
@@ -109,9 +109,11 @@ public static function rewrite()
 
         // Add Caching
         $cacheClass = Yii::$app->settings->get('cache.class');
+        $cacheUseApcu = Yii::$app->settings->get('cache.useApcu');
         if (in_array($cacheClass, ['yii\caching\DummyCache', 'yii\caching\ApcCache', 'yii\caching\FileCache'])) {
             $config['components']['cache'] = [
                 'class' => $cacheClass,
+                'useApcu' => $cacheUseApcu,
```

Comment:
no, that does not make sense, the if needs to have different branches, one for Apc and one for DummyCache and FileCache.

```php
         if (in_array($cacheClass, ['yii\caching\DummyCache', 'yii\caching\FileCache'])) {
             $config['components']['cache'] = [
                 'class' => $cacheClass,
                 'keyPrefix' => Yii::$app->id
             ];
         } elseif ($cacheClass === 'yii\caching\ApcCache') {
             $config['components']['cache'] = [
                 'class' => $cacheClass,
                 'useApcu' => $cacheUseApcu,
                 'keyPrefix' => Yii::$app->id
             ];
         }
```